src/components/container/AIAssistant/AIAssistant.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { FontAwesomeSvgIcon } from 'react-fontawesome-svg-icon';
+import { faFlask } from '@fortawesome/free-solid-svg-icons/faFlask';
+import { StreamEvent } from '@langchain/core/tracers/log_stream';
+import { AIMessageChunk } from '@langchain/core/messages';
+import { StructuredTool } from '@langchain/core/tools';
+import MyDataHelps from '@careevolution/mydatahelps-js';
+
+import { MyDataHelpsAssistant } from '../../../helpers/assistant/assistant';
+import Chat from '../../presentational/Chat';
+
+import '@fortawesome/fontawesome-svg-core/styles.css';
+
+export interface AIAssistantProps {
```

Comment:
I think we should have an initial message string prop that populates in the chat as the first message, so users see something when they land on the page...e.g. "Ask questions about your lab reports" or something

Reply:
I would like to leave this as an upcoming improvement to offer a few rotating suggestions just like chat gpt for interesting prompts:

![image](https://github.com/user-attachments/assets/0a38b1ed-ff8c-462d-8b48-71dcc464da4d)

Reply:
Interesting.  Yeah it's not super critical since we can throw a text block above it